packages/internal-playwright-helpers/testApp.ts

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright 2022 Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import { Page } from "@playwright/test";
+import { CognitoPage } from "./cognito";
+import { OpenIdPage } from "./open-id";
+import { getBrowserTestingEnvironment } from "@inrupt/internal-test-env";
+import {
+  TESTID_OPENID_PROVIDER_INPUT,
+  TESTID_LOGIN_BUTTON,
+  TESTID_ERROR_MESSAGE,
+  TESTID_SESSION_STATUS
+} from "@inrupt/internal-playwright-testids";
+
+export class TestPage {
+  page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  async start() {
+    await this.page.goto("/");
+  }
+
+  private async startLogin() {
+    const { idp } = getBrowserTestingEnvironment();
```

Comment:
We can pass this into the constructor for TestPage

Reply:
https://github.com/inrupt/typescript-sdk-tools/pull/78/commits/0e79133024ded1b6dccb0e8e7cda407111cefcd7